datafusion/sql/src/utils.rs

```diff
@@ -535,16 +535,8 @@ pub(crate) fn make_decimal_type(
     }
 }
 
-// Normalize an identifier to a lowercase string unless the identifier is quoted.
-pub(crate) fn normalize_ident(id: &Ident) -> String {
-    match id.quote_style {
-        Some(_) => id.value.clone(),
```

Comment:
here the value is always cloned which is not necessary for most uses when we already have an owned string